app/controllers/offences_controller.rb

```diff
@@ -22,8 +22,7 @@ def update
 
   def update_document_workflow
     workflow = DocumentWorkflow.new(offences)
-    workflow.update_status(:complete) ||
-      workflow.update_status(:incomplete)
+    workflow.update_status(:complete)
```

Comment:
This could maybe user a comment to highlight that _currently_ the Offences document is considered to be complete if the form passes validation.